tox.ini

```diff
@@ -31,40 +41,56 @@ basepython =
     pypy3: pypy3
 
 deps =
-    test: coverage==4.5.4
     test: idna==2.8
     test: typing==3.7.4.1
     test: {py26,py27,py34}: pytest==4.6.7
     test: {py35,py36,py37,py38}: pytest==5.2.4
     test: pytest-cov==2.8.1
+    test: {[testenv:coverage_report]deps}
```

Comment:
Use coverage version defined by the `coverage_report` environment. This reduces duplication of this info (and happens to make requires.io work correctly).